geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ExportStackTraceCommandDUnitTest.java

```diff
@@ -97,4 +104,40 @@ public void exportStackTrace_with_file() {
     // delete this file afterwards so that we won't pollute the other tests in this class
     files[0].delete();
   }
+
+  @Test
+  public void exportStackTrace_check_file_content() throws IOException {
+    File stackTraceFile = new File(locator.getWorkingDir(), "my_file");
+    String stackTraceMsg = "*** Stack-trace for member server-1 at ";
+    boolean containProperlyMsg = false;
+
+    gfsh.executeAndAssertThat("export stack-traces --file=" + stackTraceFile.getAbsolutePath())
+        .statusIsSuccess().containsOutput("stack-trace(s) exported to file");
+
+    Date date = new Date();
+    String possibleMsg = stackTraceMsg + new SimpleDateFormat("yyyy/MM/dd HH:mm:ss").format(date);
+    date.setSeconds(date.getSeconds() - 1);
+    String anotherPossibleMsg =
+        stackTraceMsg + new SimpleDateFormat("yyyy/MM/dd HH:mm:ss").format(date);
+
+    List<String> expectedStrings = Arrays.asList(possibleMsg, anotherPossibleMsg);
```

Comment:
This will probably introduce flakiness in the tests as there's no guarantee that the `Date` created in the test will match (or be 1 second apart) from the one used in the actual command.
You'd need to use another approach to verify the changes; maybe check that the stack trace for each member **_contains_** a date and that the date follows the pattern you use within the command (`yyyy/MM/dd HH:mm:ss`)?.